kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -195,8 +209,48 @@ server {
         }
     }
 
+    location @error_400 {
+        default_type application/json;
+        return 400 '{"error_message": "Bad request", "error_code": "bad-request"}';
+    }
+
     location @error_401 {
         default_type application/json;
         return 401 '{"error_message": "Access denied.", "error_code": "unauthorized"}';
     }
+
+    location @error_403 {
+        default_type application/json;
+        return 403 '{"error_message": "Not permission for this page", "error_code": "forbidden"}';
```

Comment:
Are these all errors that nginx can return? What about the ones not listed here? Is there a generic mechanism that can let us replace all the errors so that we're sure no non-JSON errors ever get returned?

Reply:
Well, I don't find alternative solution to replace all non-JSON errors. I think that all most common errors are covered. In `nginx-storage` config are more errors included, which is cover possible errors related to uploading files. Are we want cover all possible error ?

Reply:
>  In nginx-storage config are more errors included

Which one specifically?